samples/batch/js/node/main.js

```diff
@@ -0,0 +1,333 @@
+// Your subscription key and region for the speech service
+const SUBSCRIPTION_KEY = ""
+
+const SERVICE_REGION = "centralindia"
+
+const NAME = "Simple transcription"
+const DESCRIPTION = "Simple transcription description"
+
+const LOCALE = "en-US"
+const RECORDINGS_BLOB_URI = ""
+
+// Provide the uri of a container with audio files for transcribing all of them with a single request
+const RECORDINGS_CONTAINER_URI = ""
+
+//Set model information when doing transcription with custom models
+const MODEL_REFERENCE = 'None' ; // guid of a custom model
+
+var callback2 = function(error, data, response) {
+        var completed = "False";
+        if (error) {
+          console.error(error);
+        } else {
+          console.log('API called successfully. Returned data ' );
+          var jsonParsed = JSON.parse(JSON.stringify(data));
+          console.log(jsonParsed.status);
+          var trans_status = jsonParsed.status;
+
+          if (trans_status == "Failed" || trans_status == "Succeeded")
+          {
+            completed = "True"
+          }
+        if (trans_status == "Succeeded")
+          {
+           
+          }
+        else if (trans_status == "Failed")
+            console.log("Transcription failed: {transcription.properties.error.message}")
+
+        }
+      };
+
+ var callback = function(error, data, response) {
+  if (error) {
+    console.error(error);
+  } else {
+   console.log('API called successfully. Returned data: ' + JSON.stringify(data));
+
+  }
+};
+
+var callbackGetTrans = function(error, data, response) {
+  if (error) {
+    console.error(error);
+  } else {
+   console.log('Callback Get API called successfully. Returned data: ' + JSON.stringify(data));
+    completed=false ;
+    var tid = data.self.substring(data.self.lastIndexOf("/")+1);
+    console.log("Trans Id " + tid);
+    status = _checkComplete(tid);
+
+    if (status == "Succeeded")
+    {
+        var opts = _getTranscriptionFiles();
+        apiInstance.getTranscriptionFiles(tid,opts,callbackGetTransFiles);
+    }
+    else if(data.status == "Failed")
+           console.log("Transcription failed: {transcription.properties.error.message}")
+  }
+};
+
+var callbackGetTransFiles = function(error, data, response) {
+  if (error) {
+    console.error(error);
+  } else {
+   console.log('callbackGetTransFiles API called successfully. Returned data: ' + JSON.stringify(data));
+   pag_files = data.values;
+   console.log(pag_files.length);
+   for (var i = 0; i < pag_files.length; i++){
+     var file_data = pag_files[i];
+     var url = file_data.links.contentUrl;
+    
+    console.log (" url " + url);
+     if(file_data.kind!='Transcription')
+        continue;
+     else{
+        request(url, { json: true }, (err, res, body) => {
+            if (err) { 
+                return console.log(err); 
+            }
+         //   console.log("***************************************************************  response "+JSON.stringify(body));
```

Comment:
Please remove all commented code (except for features that are optional in the sample)

Reply:
@chlandsi - Have incorporated the changes in the latest code for main.js.